src/Modules/Shared/PowerShellGet/PSModule.psm1

```diff
@@ -30,23 +30,30 @@ else
 
 if(IsWindows)
 {
-    try
-    {
-        $script:MyDocumentsFolderPath = [Environment]::GetFolderPath("MyDocuments")
-    }
-    catch
+    if(IsCoreCLR)

```

Comment:
I think this check isn't quite correct. This should be an "if inbox powershell" check. Inbox PowerShell for full and core CLRs should use the path on line 41. Non-inbox PowerShell should use the $HOME path on line 35.